doc/cosign_version.md

```diff
@@ -21,6 +21,9 @@ cosign version [flags]
   -d, --verbose              log debug output
 ```
 
+## Expected Behaviour
```

Comment:
These docs are autogenerated, so either you need to add this text to the version CLI or put this in another docs file.

Reply:
That clarifies the documentation sturcture. I'll add another markdown file just for the expected behaviour of the `version` command.